editor/editor_node.cpp

```diff
@@ -851,7 +851,7 @@ void EditorNode::_notification(int p_what) {
 
 			command_palette->register_shortcuts_as_command();
 
-			callable_mp(this, &EditorNode::_begin_first_scan).call_deferred();
+			EditorNode::_begin_first_scan();
```

Comment:
First scan should be called as soon as possible, delaying it causes issues when flushing the MessageQueue.